utils/notifications.tsx

```diff
@@ -14,7 +14,29 @@ let requestedNotificationPermission = false;
 // notification. Notifications that do not require interaction will be closed automatically after
 // the Constants.DEFAULT_NOTIFICATION_DURATION. Not all platforms support all features, and may
 // choose different semantics for the notifications.
-export async function showNotification({title, body, requireInteraction, silent, onClick} = {}) {
+
+export interface ShowNotificationParams {
```

Comment:
We are using type aliases for this cases, probably something more consistent would be `type Props = {...`, besides you are exporting but there is no need to export it.

Reply:
Nevermind, this is correct, because is not a component, is a function.